### PR TITLE
Remove hard coded covid hotfixes

### PR DESCRIPTION
**Pull request by stanislaw-zakrzewski**

### CODE REVIEW

File: dev/make_superuser.sh [hunk 1]
```diff
@@ -8,7 +8,7 @@ pushd `pwd`
 cd `dirname "$0"`
 
 # Tell us what database to use — default is covid19 but you can override it for other instances
```
Comment (abhidg) [inline dev/make_superuser.sh:10]: update comment

File: verification/curator-service/ui/cypress/e2e/components/AppTest.spec.ts [hunk 2]
```diff
@@ -6,43 +6,43 @@ describe('App', function () {
     beforeEach(() => {
         cy.task('clearSourcesDB', {});
         cy.seedLocation({
-            country: 'DEU',
+            country: 'DE',
```
Comment (abhidg) [inline verification/curator-service/ui/cypress/e2e/components/AppTest.spec.ts:9]: Are we going with three letter or two letter codes? I thought we had switched to three letters, could be mistaken. Either is fine as long as we use the same convention everywhere

**Review by abhidg — COMMENTED** [review]
See comment about ISO2/3 codes, fine with either

File: verification/curator-service/ui/cypress/e2e/components/AppTest.spec.ts (hunk 2 above)
Comment (stanislaw-zakrzewski) [reply verification/curator-service/ui/cypress/e2e/components/AppTest.spec.ts:9]: We do switch to ISO3, but the geolocation service is working with ISO2 codes. So I have to seed it with ISO2 codes.

**Review by abhidg — APPROVED** [review]
Thanks!

**Pull request merged**